src/main/resources/languages/pl-PL.json

```diff
@@ -339,8 +339,8 @@
       "description": "Zbanuj podanego gracza."
     },
     "unlockall": {
-      "success": "🇺🇸Unlocked all open states for %s.",
-      "description": "🇺🇸Unlocks all open states for a player."
+      "success": "🇺🇸Odblokowano wszystkie stany dla %s.",
+      "description": "🇺🇸Odblokowuje wszystkie stany."
```

Comment:
```suggestion
      "success": "Odblokowano wszystkie stany dla %s.",
      "description": "Odblokowuje wszystkie stany."
```

Reply:
Oh so that was this weird us, in idea it was shown as just "us" and I wasn't sure what that was so I just left this